atomic_reactor/plugins/exit_koji_import.py

```diff
@@ -361,6 +361,14 @@ def get_build(self, metadata, worker_metadatas):
             except ValueError:
                 self.log.error("invalid task ID %r", koji_task_id, exc_info=1)
 
+        isolated = metadata.get('labels', {}).get('isolated', False)
+        if isolated is not None:
```

Comment:
Not sure if `isolated` can ever be `None`, `get` would set it to `False`. Just `if isolated` should be sufficient. `try..catch` is not necessary as well

Reply:
I'm not sure I agree on this. What if we get a bad value from OSBS?

Reply:
Good point. Maybe `.get()` is not necessary then:
```
try:
    isolated = str(metadata['labels']['isolated']).lower() == 'true'
except IndexError:
    isolated = False
```

`IndexError` would guard against missing keys and `AttributeError` would ensure the returned value (forcibly converted into string) is True. This way even a string would be converted to a boolean, although I'm not sure if its intended.

In any case `bool("False")` would not throw exception and evaluate into `True`.

Reply:
Does this cover cases the current code doesn't cover? If not, I'd advocate for leaving it as-is just because I feel that this is hard to read.

Reply:
It does - if `isolated` there is `foo` it would evaluate into `True` with current code.

Suggested code is a bit complex, yes, maybe @lcarva has ideas about improving it.

Reply:
Hmm, you're right about the 'foo' case. String 'False' would result in True too.

Reply:
A lot of unit tests don't set 'labels' so I added an exception for KeyError.

Reply:
We expect that value to be a boolean. If it's not, let's treat it as an error instead of trying to mangling it into a string...
IMO opinion the original version of this code was the easiest to follow.

Reply:
Okay, agree. `bool(isolated)` doesn't do anything useful though